exercises/C-comparison-operators/exercise.js

```diff
@@ -7,14 +7,14 @@
 
 let studentCount = 16;
 let mentorCount = 9;
-let moreStudentsThanMentors; // finish this statement
+let moreStudentsThanMentors = studentCount>mentorCount; // finish this statement
 
 let roomMaxCapacity = 25;
-let enoughSpaceInRoom; // finish this statement
+let enoughSpaceInRoom = studentCount+mentorCount<=roomMaxCapacity; // finish this statement
```

Comment:
```suggestion
let enoughSpaceInRoom = studentCount + mentorCount <= roomMaxCapacity; // finish this statement
```

Reply:
Hi Saadia, one thing I would suggest is sometimes you don't leave spaces which reduces how readable your code is, which is a shame because it's really neat and efficient.